client/config.go

```diff
@@ -23,7 +23,13 @@ func InitConfig(cmd *cobra.Command) error {
 	}
 
 	configFile := path.Join(home, "config", "config.toml")
-	if _, err := os.Stat(configFile); err == nil {
+	_, err = os.Stat(configFile)
```

Comment:
It is still incorrect! Should be os.IsNotExist
![image](https://user-images.githubusercontent.com/4898263/132665865-4eb8efe3-2f13-499b-82c6-42461bc547f5.png)

Reply:
Done